db/migrate/20150728163945_create_cards.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCards < ActiveRecord::Migration
+  def change
+    create_table :cards do |t|
+    	t.string			:original_text
+    	t.string			:translated_text
+    	t.datetime		:review_date
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Tab detected.